### DISCUSSION TITLE
Cannot renew GitHub Education - I'm not near the campus location. I'm Ukrainian, currently in Europe for obvious reasons

### DISCUSSION THREAD

**Discussion by denbondd** [community/community#67807] [GitHub Education]
### Hi everyone,

Hello, I'm a citizen of Ukraine and a student in Ukrainian University, signed up for GitHub education hub in my first year in Uni (2021).
Now I need to renew this subscription. I had to move to Spain, Europe, because of the war. But I'm still studying in that same Ukrainian University remotely (everything is remote there now). 
So, when I'm trying to renew the subscription with my Student ID and University email, I get the following errors:

``` 
The item you uploaded is insufficient to demonstrate your current academic status.
Please see suggestions.
Suggested proof to expedite your approval:

The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote. If you believe that the school you have applied for has incorrect information that affects your current application e.g. incorrect email domain, please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket.

You appear not to be near any campus location for the school you have selected. Your school-provided academic affiliation document must indicate virtual learning if you are remote.Please enable the application to discover your location from the browser pop-up to provide an accurate location. If you are using a VPN, please disable it when you reapply. If you believe that the school you have applied for has incorrect information that affects your current application e.g. incorrect campus location, please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket. 
```

Which is completely understandable since my University doesn't have campus in Spain. But does it mean that I won't be able to get the Student Pack? It was really helping me for the past 2 years.

**Comment by yarikLev04** [comment]
Have the same problem. I'm a Ukrainian and signed up for GitHub education hub in 2021, it was my first year of university. But now I can't renew my subscription, I have moved in Slovakia because of the war and all universities in Ukraine work remotely and don't have any campuses in other countries

**Comment by StasonMendelso** [comment]
Hello.  I had the same problem about a year ago, but I was in Ukraine in a different city.  I contacted support, but was ignored.  Why is there no answer?
